src/main/java/org/testeditor/fixture/web/io/FileReader.java

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.web.io;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testeditor.fixture.web.WebDriverFixture;
+
+import com.google.common.io.CharStreams;
+
+public class FileReader {
+    
+    private String fileName;
+    private static final Logger logger = LoggerFactory.getLogger(WebDriverFixture.class);
+
+    public FileReader(String fileName) {
+        this.fileName = fileName;
+    }
+    
+
+    /**
+     * Opens a file defined as fileName and read the content line by line.
+     * 
+     * @return File content as String
+     */
+    public String getFileContentAsString() {
+
+        // Get file from resources folder
+        ClassLoader classLoader = getClass().getClassLoader();
+        InputStream jsonResource = classLoader.getResourceAsStream(fileName);
+       
+        String result = null;
+        try {
+            result = CharStreams.toString(new InputStreamReader(jsonResource, StandardCharsets.UTF_8));
+        } catch (IOException e) {
+            logger.error("The file with the name {} can not be read in the resource folder", fileName);
```

Comment:
how about logger.error("..." + fileName + "...", e)
puts the stack trace into the log (whereas printStackTrace will print to stdout, regardless of log configuration)